data/helpers/users.js

```diff
@@ -0,0 +1,23 @@
+const db = require("../dbConfig.js");
+
+module.exports = {
+  addUser,
+  findUser,
+  getUsers
+};
+
+function addUser(user) {
+  return db("users").insert(user);
+}
+
+function findUser(user) {
+  return db("users")
+    .where({
+      name: user.name
```

Comment:
You could alternatively [destructure](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) the name property off of the user object passed in here in order to use the object shorthand (where a key has the same name as a variable holding its corresponding value):
```javascript
function findUser({ name }) {
  return db("users")
    .where({ name })
    .first();
}
```